engine/src/io/DataLoader.cpp

```diff
@@ -149,7 +148,7 @@ std::unique_ptr<ral::frame::BlazingTable> data_loader::load_data(
 					}
 				} else {
 					Library::Logging::Logger().logError(ral::utilities::buildLogString(
-						"", "", "", "ERROR: Was unable to open " + user_readable_file_handles[file_index]));
+						"", "", "", "ERROR: Was unable to open " + file_sets[file_set_index][file_in_set].uri.toString()));
```

Comment:
There was a reason to keep this seperate before. Because the handle we have is NOT the same as what the user registered it as on their side when they made the filesystem

Reply:
user_readable_file_handles[file_index] in this context and file_sets[file_set_index][file_in_set].uri.toString() are created the exact same way, I am not sure what you are talking about. You may be thinking of how we would like for this to work, not of how it was actually working